requirements_dev.txt

```diff
@@ -1,5 +1,9 @@
+ipython
+m2r2
```

Comment:
It would be better to make a `requirements_doc.txt`, as none of this is required for development.

Equally, `requirements_doc.txt` can leave out pytest, mysql-connector-python, etc.

Reply:
I've now moved these requirements to a separate `docs/requirements.txt`